games-misc/parsec-bin/parsec-bin-147.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit gnome2-utils unpacker
+
+PARSEC_P="parsec-bin-${PV}"
+
+DESCRIPTION="Simple, Low-Latency Game Streaming - Play, watch, share games with your friends"
+HOMEPAGE="https://parsecgaming.com/"
+SRC_URI="https://s3.amazonaws.com/parsec-build/package/parsec-linux.deb -> $PARSEC_P.deb"
```

Comment:
Add curly brackets, `${PARSEC_P}` here.

Reply:
Right, i don't check before to push the commit. My fault. :-)